### DISCUSSION TITLE
Livereload not reflect change

### DISCUSSION THREAD

**Discussion by apomili** [wardenenv/warden#603] [Help & Support]
I guys, I'm trying to use LiveReload on a Magento 2 env.
I followed this guide: https://docs.warden.dev/configuration/livereload.html

Everything seems OK but when I save a less file the browser not reflect the change until do a manual f5.

I tested it on chrome and firefox with same result. 

In chrome I used this plugin in https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei

This is a network snapshot with the correct request for livereload 

![Schermata del 2022-09-27 11-38-11](https://user-images.githubusercontent.com/7668956/192491870-a0106973-1a78-4457-9858-2105eac2f096.png)

**Comment by mattkrupnik** [comment]
@apomili try to use `usePolling` in watch segment in your gulpfile.js for example `gulp.watch([path + '**/*.less'], {usePolling: true}, gulp.series('less'));`

**Comment by apomili** [comment]
Hi Matt, thx for the reply.

I'm using grunt task as magento doc describes (https://developer.adobe.com/commerce/frontend-core/guide/css/debug/). 
I tried to add your options in the after the lines 30 and 38 of this files with no effect.
https://github.com/magento/magento2/blob/2.4-develop/dev/tools/grunt/configs/watch.js

But my suspect is that the livereload plugin can't reach the server cause if I click twice the liverload icon on browser (first click to stop and second to reconnect) I got the error "Could not connect to LiveReload Server"
